src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -67,6 +153,32 @@ public JsonElement GetProperty(ReadOnlySpan<char> propertyName)
             throw new KeyNotFoundException();
         }
 
+        /// <summary>
+        ///   Gets a <see cref="JsonElement"/> representing the value of a required property identified
+        ///   by <paramref name="utf8PropertyName"/>.
+        /// </summary>
+        /// <remarks>
+        ///   Property name matching is performed as an ordinal, case-sensitive, comparison.
+        ///
+        ///   If a property is defined multiple times for the same object, the last such definition is
+        ///   what is matched.
+        /// </remarks>
+        /// <param name="utf8PropertyName">
+        ///   Name of the property whose value to return encoded as UTF-8 with no Byte-Order-Mark (BOM).
```

Comment:
😕 This sentence is ambiguous: it could be saying the argument is UTF-8 encoded, or saying the return value is UTF-8 encoded. It would help to reword this to eliminate the ambiguity.